libs/csm/rest/csm_rest_system_health.py

```diff
@@ -626,3 +626,54 @@ def cluster_operation_signal(
         self.log.info("%s operation on %s POST REST API response : %s",
                       operation, resource, response)
         return True, response
+
+    @RestTestLib.authenticate_and_login
+    @RestTestLib.rest_logout
```

Comment:
Why logout? We have to keep session alive right?

Reply:
> Why logout? We have to keep session alive right?

Then we need to add one more variable to keep the header value handy. And same we need to use while getting status of resource. 

Dose the same logic will be appiciable to control pod also by setting and getting control pod status ?

Reply:
For set, need to check but it is needed for get for control pod.

Reply:
Then for get, as we are expecting its status to be online only, it should allow us to do new login also